src/main/kotlin/no/nav/familie/ef/sak/brev/VedtaksbrevService.kt

```diff
@@ -95,12 +120,42 @@ class VedtaksbrevService(private val brevClient: BrevClient,
 
         validerBeslutterIkkeErLikSaksbehandler(besluttervedtaksbrev)
 
-        val beslutterPdf = Fil(brevClient.genererBrev(besluttervedtaksbrev.tilDto(signaturMedEnhet.skjulBeslutter)))
+        val beslutterPdf = lagBeslutterPdfMedSignatur(besluttervedtaksbrev, signaturMedEnhet)
+
         val besluttervedtaksbrevMedPdf = besluttervedtaksbrev.copy(beslutterPdf = beslutterPdf)
         brevRepository.update(besluttervedtaksbrevMedPdf)
         return beslutterPdf.bytes
     }
 
+    private fun validerBehandlingKanBesluttes(behandling: Behandling) {
+        if (behandling.steg != StegType.BESLUTTE_VEDTAK || behandling.status != BehandlingStatus.FATTER_VEDTAK) {
+            throw Feil("Behandling er i feil steg=${behandling.steg} status=${behandling.status}",
+                       httpStatus = HttpStatus.BAD_REQUEST)
+        }
+    }
+
+    private fun lagBeslutterPdfMedSignatur(besluttervedtaksbrev: Vedtaksbrev,
+                                           signaturMedEnhet: SignaturDto) =
+            when (besluttervedtaksbrev.saksbehandlerHtml != null) { // TODO: saksbehandlerHtml skal kanskje bli ikke-nullable.
```

Comment:
Hvordan skal vi få til det? Er det mulig å patche gamle brev eller blir det litt tvilsomt siden brevmalene er endret med tiden?

Reply:
Det blir nog vanskelig 😬  men vi kan kaste feil senere hvis den er null, ettersom det ikke skal skje når alle behandlinger som har den gamle typen har blitt behandlet... Men det kan nog ta laaaaaang tid, har ju noen behandlinger som blir liggendes i måneder, men ikke sikkert alle de har brev

Reply:
Vil tro det ikke er så mange som ligger til beslutter i ukevis? Sett en reminder om 4 uker og sjekk DB-innhold og endre til å kaste feil her og nedenfor?

Reply:
Ser poenget ditt, og ikke sikkert vi vil gjøre det. Men etter at et brev er besluttet vil vi aldri bruke html (eller saksbehandlerBrevrequest), så vi kan kanskje sette alle gamle brev til en "Ikke satt verdi" for å slippe å håndtere null diverse steder.

Reply:
Tror ikke vi må bente så lenge til alle brev som er sendt til beslutter er ute av pipa, så det skal nok gå bra. 
Bare drodling: Kan sette (patche) gamle brevrequest til tom streng, eller liknende - endre til not null, og kaste feil hvis "tom streng" - hadde vært fint å få en not-nullable på denne. 

Uansett det blir noe vi må tenke på til i neste pr :)